### DISCUSSION TITLE
Changelog and/or release description

### DISCUSSION THREAD

**Discussion by andreas-bulling** [revk/xESP32-Faikout#205] [Ideas]
Hi,

would it be possible to maintain a changelog and/or provide (short/bullet list like) descriptions for each release? 

It's currently difficult, if not impossible, to know what has changed, what has been fixed and - most importantly - which new functionality has been added.

For example, I just upgraded the firmware on my four indoor units and noticed the new "Demand" slider. But I have no idea what it does...

**Comment by revk** [comment]
https://github.com/revk/ESP32-Faikin/wiki/Work-in-progress
